apps/v1/types.go

```diff
@@ -14,6 +14,9 @@ import (
 // +genclient:method=GetScale,verb=get,subresource=scale,result=k8s.io/api/extensions/v1beta1.Scale
 // +genclient:method=UpdateScale,verb=update,subresource=scale,input=k8s.io/api/extensions/v1beta1.Scale,result=k8s.io/api/extensions/v1beta1.Scale
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +k8s:prerelease-lifecycle-gen:introduced=1.6
```

Comment:
I am not exactly sure when DC api types were introduced. Picking 1.6.

Reply:
How important do you think it is that this is correct?

Given we only really care about this for openshift 4, would it be sensible to pick whatever K8s version the first OCP 4 release was on? I think 4.0 (never release?) would have been 1.13 right?

Reply:
> How important do you think it is that this is correct?

It does not have to be scientifically accurate :) 4.0 = 1.13 works as well :)